src/vmm/src/persist.rs

```diff
@@ -249,7 +249,10 @@ fn snapshot_memory_to_file(
                 .dump_dirty(&mut file, &dirty_bitmap)
                 .map_err(Memory)
         }
-        SnapshotType::Full => vmm.guest_memory().dump(&mut file).map_err(Memory),
+        SnapshotType::Full => {
+            let _ = vmm.get_dirty_bitmap().map_err(DirtyBitmap)?;
```

Comment:
I think we should only get the bitmap if it exists (see CI errors). Probably by handling the error here.